src/app/reports-handler.ts

```diff
@@ -82,6 +83,11 @@ export const exportLogs = (): void => {
         });
         return;
     }
+
+    for (const log of logs) {
+        fs.writeFileSync(source + ( !isMac && !isLinux ? '\\' : '' ) + log.filename + '.log', log.contents );
```

Comment:
We can use `path.join` instead of this ` !isMac && !isLinux ? '\\' : '' `

Reply:
Good idea, I will change it to that :)